src/main/scala/beam/utils/scripts/FailFast.scala

```diff
@@ -34,5 +34,11 @@ object FailFast extends LazyLogging {
       )
     }
 
+    if (beamServices.beamConfig.beam.agentsim.agents.rideHail.allocationManager.requestBufferTimeoutInSeconds != 0 &&
+        beamServices.beamConfig.beam.agentsim.schedulerParallelismWindow > beamServices.beamConfig.beam.agentsim.agents.rideHail.allocationManager.requestBufferTimeoutInSeconds) {
+      throw new RuntimeException(
+        "Scheduler Parallelism Window must be greater then Request Buffer Timeout"
```

Comment:
must be smaller than

Reply:
change comparision from greater then to less then